File: TypeScript/codewars/Typescript/Multiplication-table.ts
export function multiplicationTable(size: number): number[][] {
	const output: number[][] = [];

	for (let i = 1; i <= size; i++) {
		const row: number[] = [];
		for (let j = 1; j <= size; j++) {
			row.push(i * j);
		}
		output.push(row);
	}

	return output;
}

File: Csharp/CodeWars/src/Completed/MultiplicationTable.cs
namespace CodeWars.Completed;

public class MultiplicationTableClass
{
    public static int[,] MultiplicationTable(int size)
    {
        int[,] output = new int[size, size];

        for (int i = 0; i < size; i++)
        {
            for (int j = 0; j < size; j++)
            {
                output[i, j] = (i + 1) * (j + 1);
            }
        }

        return output;
    }
}
